proprietary/tokens/src/components/amsterdam/grid.tokens.json

```diff
@@ -3,13 +3,20 @@
     "grid": {
       "column-count": { "value": "4" },
       "gap": {
```

Comment:
Maybe we should separate `column-gap` and `row-gap`? `column-gap` only needs one value, makes it a bit easier to follow maybe.

Reply:
Now it’s my turn to think this would be overcomplicating things.

The CSS `gap` property is a shorthand that sets the same value for both `column-gap` and `row-gap`. I think that’s what we want: the same options for both horizontal and vertical spacing. I wouldn’t expect the community to need two different scales, either.

Is it worthwhile to have both `amsterdam-grid-column-gap` and `amsterdam-grid-row-gap` with the same options?